File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/run_step_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class RunStepResponse:
    """OpenAI run step object"""
    id: str = None
    object: str = None
    type: str = None
    status: str = None
    metadata: Dict[str, str] = None
    created_at: int = None
    assistant_id: str = None
    thread_id: str = None
    run_id: str = None
    step_details: Any = None
    last_error: Any = None
    expired_at: int = None
    cancelled_at: int = None
    failed_at: int = None
    completed_at: int = None

File: sdkwork-ai-sdk-go/types/run_step_response.go
package types

// OpenAI run step object
type RunStepResponse struct {
	Id string `json:"id"`
	Object string `json:"object"`
	Type string `json:"type"`
	Status string `json:"status"`
	Metadata map[string]string `json:"metadata"`
	CreatedAt int `json:"created_at"`
	AssistantId string `json:"assistant_id"`
	ThreadId string `json:"thread_id"`
	RunId string `json:"run_id"`
	StepDetails interface{} `json:"step_details"`
	LastError interface{} `json:"last_error"`
	ExpiredAt int `json:"expired_at"`
	CancelledAt int `json:"cancelled_at"`
	FailedAt int `json:"failed_at"`
	CompletedAt int `json:"completed_at"`
}
